### PR TITLE
Feat/swap tiers messages

### PR DESCRIPTION
**Pull request by tony-blockchain**
## Description
Add a concise explanation of the changes.

## Change Type
Please enter one or more of the following: 
- Feature
- Bug Fix
- CI/CD
- Upgrades
- Other (add explanation)

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## Code Checklist
- [ ] Code compiles successfully (verified via `yarn start`)
- [ ] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/MinMaxButtons/MinMaxButtons.js [hunk 1]
```diff
@@ -6,25 +6,29 @@ import { connect } from 'react-redux'
...
 export const MinMaxValue = styled.div`
-  font-weight: 600;
+  font-weight: 500;
   font-size: 14px;
 `
-
-const formatAmount = (isFiat, symbol, value) =>
-  isFiat ? `${symbol}${value}` : `${value} ${symbol}`
-
+const MixMaxRow = styled(Row)`
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/MinMaxButtons/MinMaxButtons.js]: MinMax?

Comment (tony-blockchain) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/MinMaxButtons/MinMaxButtons.js]: yes, the row with min & max butons

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/MinMaxButtons/MinMaxButtons.js]: I think its a small copy MinMax

Comment (tony-blockchain) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/MinMaxButtons/MinMaxButtons.js]: ah ok, will fix in a sec

File: packages/blockchain-wallet-v4-frontend/src/components/OrderStatus/index.js [hunk 2]
```diff
@@ -128,30 +128,30 @@ export const OrderNote = ({ status }) => {
...
           defaultMessage='Thanks for placing your trade! Funds will usually reach your wallet within 2 hours and we’ll send you a notification when that happens. Keep track of your trade’s progress in the Order History tab.'
         />
       )
     case PENDING_REFUND:
       return (
         <FormattedMessage
-          id='components.orderstatus.note.refunding'
+          id='components.orderstatus.note.swaprefunding'
           defaultMessage='We are now processing a refund for any funds broadcast minus the network fee. Please check your Order History to check the refund status.'
         />
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/OrderStatus/index.js]: Not a big deal, but changing the id of a message will require new strings to be translated. If the default message doesn't change its a bit unnecessary.

File: packages/blockchain-wallet-v4-frontend/src/data/analytics/sagas.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ export default ({ api }) => {
       if (test(/cash/, text)) return yield call(api.logClick, 'bch')
       if (test(/stellar/, text)) return yield call(api.logClick, 'xlm')
       if (test(/(buy|sell)/, text)) return yield call(api.logClick, 'buysell')
-      if (test(/exchange/, text)) return yield call(api.logClick, 'exchange')
+      if (test(/swap/, text)) return yield call(api.logClick, 'swap')
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/analytics/sagas.js]: Can we leave the (api.logClick, 'exchange') and go over to swap when we add in the new service, which we can discuss offline?

**Pull request merged**